addOns/pscanrules/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Depends on an updated version of the Common Library add-on.
 - Clarified details of the Viewstate scan rule alerts, in some instances they were misleading (containing colons suggesting further data).
+- The Open Redirect scan rule's (ID: 10028) alerts have been renamed "Off-site Redirect" as this is a passive rule which compares the authority of the origin and destination and there is no assurance of a truly "open" redirect.
```

Comment:
The scan rule as well not just the alerts.

Reply:
For consistency's sake and easier future maint should I rename the class(es) and resource keys as well?

Reply:
All the "user controlled" have the same issue, maybe we should rename them all.